### DISCUSSION TITLE
Share experience with windows instalation ... help for beginners :)

### DISCUSSION THREAD

**Discussion by vmanzanal** [modelcontextprotocol/.github#50] [Q&A] [answered]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [ ] Protocol Specification
- [ ] SDK Usage
- [ ] Server Implementation
- [X] General Implementation
- [ ] Documentation
- [ ] Other

### Your Question

# Setting up MCP SQLite in Claude Desktop - Step by Step Guide

A collaborative debugging journey between Claude and a beginner user 😊

## Initial Problem
- Followed the MCP quickstart guide
- Correctly set up database and json configuration
- The wrench icon (🔧) wasn't showing up
- No database connection was established

## Solution
1. **Critical Step**: Enable Developer Mode
   - Open Claude Desktop
   - Go to "Developer" menu
   - Enable "Developer Mode"
   - MCP server starts automatically

2. **Correct Configuration**
   - File name: `claude_desktop_config.json` (not config.json)
   - Location: `C:\Users\[YOUR_USERNAME]\AppData\Roaming\Claude\`
   - Configuration example:
   ```json
   {
       "mcpServers": {
           "sqlite": {
               "command": "uvx",
               "args": [
                   "mcp-server-sqlite",
                   "--db-path",
                   "C:\\Users\\YOUR_USERNAME\\test.db"
               ]
           }
       }
   }
   ```

3. **Verification**
   - The wrench icon (🔧) appears with 6 options
   - Claude can access the database
   - MCP server starts automatically

## Important Notes
- Developer mode is a crucial step that isn't clearly documented
- The exact location and name of the configuration file are important
- No need to manually start the MCP server once developer mode is enabled

This solution was found through collaborative debugging between Claude and a user, demonstrating how AI assistants can help troubleshoot setup issues effectively.

**Comment by TimHuey** [comment]
Even enabling developer mode would not get the  tool UI element to show in the lower right corner of the prompt txtbox. I will try again and see if there has been an update to the app behavior. But it works on my mac-mini m2, just not on my windows

**Comment by TimHuey** [comment]
Nope, no updates and no changes in behavior. I can get the app to load, I get the server windows to open and spin up the sever. MCP logs show..

2024-12-01T13:36:33.597Z [info] Attempting to connect to MCP server brave-search...
2024-12-01T13:36:33.610Z [info] Connected to MCP server brave-search!

The server terminal windows shows...
Brave Search MCP Server running on stdio


Then about 10 seconds later the Claudeapp shows a popup message in the upper corner

Could not attach to MCP server brave-search 

And I can close the brave-search terminal window. It used to be locked and you couldn't close it.

**Comment by wiltonn** [comment, marked as answer]
I had same issues, wrench icon not showing up. I was using Chocolatey package manager, which seemed to mess things up. removed chocolatey and re-installed dependencies using winget seems to work.

**Reply by TimHuey** [reply]
Hmmm, no choco here on this windows. I do have winget and have used it and windows store. That's the only package managers I have.

**Comment by amira-essawy** [comment]
I didn't find the developer mode in Claude desktop and also didn't find claude_desktop_config.json
